src/lib/capture-screenshots.js

```diff
@@ -0,0 +1,27 @@
+const glob = require("glob-promise");
+const queue = require("queue");
+const path = require("path");
+
+const captureScreenshots = async () => {
+  console.log("Finding screenshot configs…");
+  const screenshotConfigs = await glob("**/*.screenshot.js");
+  console.log(`Found ${screenshotConfigs.length} screenshot configs.`);
+  const q = queue({ results: [] });
+  q.concurrency = 5;
+
+  screenshotConfigs.forEach(filepath => {
```

Comment:
I wonder if we can run all these in parallel with something like https://yarnpkg.com/package/jest-worker

Reply:
Oh wait, `queue` does this already nvm.